config/qubino/ZMNKADx.xml

```diff
@@ -0,0 +1,169 @@
+			</Help>
+            <Item label="With a Z-Wave command" value="0"/>
+            <Item label="With the capacitive input or a Z-Wave command" value="1"/>
+        </Value>
+		
+		<Value genre="config" index="10" instance="1" label="Auto on timer" units="second" max="32535" min="0" size="2" type="short" value="0">
+            <Help>
+                Default value = 0
+                0 => Disabled
+                30 to 32535 => 30 - 32535 seconds after which the device turns on.
+            </Help>
+        </Value>
+		
+        <Value genre="config" index="11" instance="1" label="Auto off timer" units="second" max="32535" min="0" size="2" type="short" value="0">
+            <Help>
+                Default value = 0
+                0 => Disabled
+                30 to 32535 => 30 - 32535 seconds after which the device turns off.
+            </Help>
+        </Value>
+		
+		<Value genre="config" index="12" instance="1" label="Auto on timer relay" units="second" max="32535" min="0" size="2" type="short" value="0">
+            <Help>
+                Default value = 0
+                0 => Disabled
+                30 to 32535 => 30 - 32535 seconds after which the device turns on.
+            </Help>
+        </Value>
+		
+        <Value genre="config" index="13" instance="1" label="Auto off timer relay" units="second" max="32535" min="0" size="2" type="short" value="0">
+            <Help>
+                Default value = 0
+                0 => Disabled
+                30 to 32535 => 30 - 32535 seconds after which the device turns off.
+            </Help>
+        </Value>
+		
+		<Value genre="config" index="30" instance="1" label="Restore state on power failure" max="1" min="0" size="1" type="list" value="1">
+            <Help>
+				With this parameter, the user can select if the state of the device (color, brightness) should be restored, after a power failure.
+				
+				Default value = 1
+				1 – Enabled (the device will restore state on power failure)
+				0 – Disabled (the device will not restore state on power failure and will remain off)
+			</Help>
+            <Item label="Disable" value="0"/>
+            <Item label="Enabled" value="1"/>
+        </Value>
+		
+		<Value genre="config" index="31" instance="1" label="Restore relay state on power failure" max="1" min="0" size="1" type="list" value="1">
+            <Help>
+				With this parameter, the user can select if the state of the built-in relay should be restored, after a power failure.
+					
+				Default value = 1
+				1 – Enabled (the device will restore state on power failure)
+				0 – Disabled (the device will not restore state on power failure and will remain off)
+			</Help>
+            <Item label="Disabled" value="0"/>
+            <Item label="Enabled" value="1"/>
+        </Value>
+		
+		<Value genre="config" index="40" instance="1" label="Watt Power Consumption Reporting Threshold for Load" units="" max="100" min="0" size="1" type="byte" value="10">

```

Comment:
Can you confirm if this is a byte or short (original was short).

Reply:
It is a byte. Thank you for fixing the issues.